colorcoder/EnumHelper.java

```diff
@@ -0,0 +1,42 @@
+package colorcoder;
```

Comment:
is EnumHelper is the valid name for the functionality encapsulated in EnumHelper.java

Reply:
My though process here is anything that helps us fetching values from the Enum as a class could be EnumHelper. What do you think?